tests/Datagrid/SimplePagerTest.php

```diff
@@ -122,4 +122,19 @@ public function testInitNoQuery(): void
         $this->expectException(\RuntimeException::class);
         $this->pager->init();
     }
+
+    public function testGetResultsAlwaysReturnsAnArray(): void
+    {
+        // phpcr odm returns ArrayCollection
+
+        $this->proxyQuery->expects($this->once())
+            ->method('execute')
+            ->with([], null)
+            ->will($this->returnValue(new ArrayCollection(\range(0, 12))));
+
+        $this->pager->setQuery($this->proxyQuery);
+        $this->pager->setMaxPerPage(2);
+
+        $this->assertSame(\range(0, 1), $this->pager->getResults());
```

Comment:
Why use the backslash here and not above?

Reply:
This is how my PS configured. it's importing functions from global scope
Based on https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/

Reply:
Why does PS import the backslash here and not above?

Reply:
Do not quite get what you mean, PS will not add backslash for old code that I did not touch.

Reply:
Oh right my bad, I saw a green line and wrongly concluded you added that line.

Reply:
if we want to be explicit, we should activate that in styleci, i think it can do that automatically. then it would be consistent over the codebase.

Reply:
ah you use php-cs-fixer, there the rule is called native_function_invocation